### DISCUSSION TITLE
Can a VDS server be used for the panel?

### DISCUSSION THREAD

**Discussion by unidentifiedddd** [UnamSanctam/UnamWebPanel#32] [Q&A]
Will there be any problems with the database?

**Comment by UnamSanctam** [comment]
No that should be fine, I host my testing panel on a VDS.

**Reply by unidentifiedddd** [reply]
> Нет, все должно быть хорошо, я размещаю свою тестовую панель на VDS.

This writes when going to the panel(
**The required SQLite PDO driver IS NOT enabled.**

**Reply by UnamSanctam** [reply]
Update your PHP.

**Reply by unidentifiedddd** [reply]
[https://i.imgur.com/OnbcTRM.png](url)

**Reply by UnamSanctam** [reply]
Update it to PHP 8 or 7.4 or try `sudo apt install php-sqlite3`
